haero/atmosphere.hpp

```diff
@@ -40,38 +70,38 @@ class Atmosphere final {
   // views storing atmospheric state data for a single vertical column
 
   /// temperature [K]
-  ColumnView temperature;
+  ConstColumnView temperature;
```

Comment:
Does this mean temperature can only be set one time? Does temperature need to be updated after a time step?

Reply:
The temperature is updated by the host model, not by us. EAMxx uses a very fine control mechanism that governs who gets to change what. It's very fancy (and pretty confusing for the uninitiated).